packages/calcite-components/src/components/list/list.tsx

```diff
@@ -254,6 +256,9 @@ export class List
    */
   @property() openable = false;
 
+  /** Specifies the size of the component. */
+  @property() scale: Scale = "m";
```

Comment:
For consistency, `scale` should be reflected.

Reply:
I made a judgment call on that because its not actually used in the css for list, but cool to add it anyway...